app/services/structure_updater.rb

```diff
@@ -26,7 +26,9 @@ def validate
     # 1. Ensure all files in the csv are present in the existing object
     csv.each.with_index(2) do |row, index|
       errors << "On row #{index} found #{row['filename']}, which appears to be a new file" unless existing_files_by_filename.key?(row['filename'])
-      errors << "On row #{index} found \"#{row['resource_type']}\", which is not a valid resource type" unless Cocina::Models::FileSetType.respond_to?(row['resource_type'])
+      unless Cocina::Models::FileSetType.properties.key?(row['resource_type'].to_sym)
```

Comment:
Will this handle 3D ok?

Reply:
Yes

Reply:
@jcoyne Sorry, my question is really more like "what values are using entering for 3D objects?" If "three_dimensional," that used to work and would now be broken. If "3d," that is broken now and will be fixed by this PR. Not something I'd block the PR over, but I am curious.

Reply:
I don't know what values would be entered. I assumed they would use `"3d"`

Reply:
@jcoyne w00t, then it's both a security enhancement and a bugfix!